ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/events/[eventName]/logs/[eventId]/EventPayload.tsx

```diff
@@ -29,15 +36,30 @@ export default async function EventPayload({ event }: EventPayloadProps) {
   const formattedPayload = JSON.stringify(parsedPayload, null, 2);
 
   return (
-    <div className="flex h-full flex-col space-y-1.5 rounded-xl bg-slate-900 text-white">
-      <header className="bg-slate-910 flex items-center justify-between rounded-t-xl p-1.5">
-        <h3 className="px-6 py-2.5">Payload</h3>
-      </header>
-      <div className="flex-1 overflow-auto">
-        <div className="p-6">
-          <SyntaxHighlighter language="json">{formattedPayload}</SyntaxHighlighter>
-        </div>
-      </div>
-    </div>
+    <DashboardCodeBlock
+      tabs={[
+        {
+          label: 'Payload',
+          content: formattedPayload,
+          language: 'json',
+          readOnly: true,
+        },
+      ]}
+      actions={
+        isSendToDevServerEnabled
+          ? [
+              {
+                label: 'Send to Dev Server',
+                title: isRunning
+                  ? 'Send event payload to running Dev Server'
+                  : 'Dev Server is not running',
+                icon: <IconLightningBolt />,
```

Comment:
Nit: Some sort of down arrow icon seems more intuitive. Maybe a down arrow inside a cloud, since the event is coming from Cloud?
![image](https://github.com/inngest/inngest/assets/20100586/adcdbd37-41a7-4a6f-886f-f403d145cf07)

Reply:
Good idea!